reference/test/solver/batch_richardson_kernels.cpp

```diff
@@ -132,7 +132,8 @@ TYPED_TEST(BatchRich, SolvesStencilSystemJacobi)
 
     auto r_1 = gko::test::solve_poisson_uniform(
         this->exec, this->solve_fn, this->opts_1, this->sys_1, 1,
-        gko::preconditioner::BatchJacobi<T>::build().on(this->exec));
```

Comment:
I think for the BatchRichardson solver, it would be good to have a block Jacobi as an inner solver. I think that will be a common use case, so we should make sure to test that.

Reply:
This now uses block jacobi.